simphony/core/keywords.py

```diff
@@ -36,7 +36,7 @@
         name='ElectrostaticField',
         definition='Electrostatic field',  # noqa
         key='ELECTROSTATIC_FIELD',
-        shape=[],
+        shape=[1],
```

Comment:
Shape of the field should be [3]. This causes an error when instantiating ConstantElectrostaticFieldModel.